app/workers/asset_manager_update_whitehall_asset_worker.rb

```diff
@@ -0,0 +1,15 @@
+class AssetManagerUpdateWhitehallAssetWorker < WorkerBase
+  sidekiq_options queue: "asset_manager_updater"
+
+  def perform(attachment_data_global_id, attributes)
+    asset_data = GlobalID::Locator.locate(attachment_data_global_id)
+    path = asset_data.file.asset_manager_path
+    AssetManager::AssetUpdater.call(asset_data, path, attributes)
```

Comment:
I assume this method can (and will) raise exceptions on some assets that don't exist. It'd probably be prudent to handle them rather than letting a unresolvable retry cycle occur.